contracts/utility/FlowEVMBridgeHandlerInterfaces.cdc

```diff
@@ -0,0 +1,131 @@
+import "FungibleToken"
+import "NonFungibleToken"
```

Comment:
why does this need to import NonFungibleToken?

Reply:
probably doesn't, but this is a dependency from the bridge smart contracts, so no need to worry about it here